src/powerhouse/components/tree-view-item/tree-view-item.tsx

```diff
@@ -46,14 +49,24 @@ export const TreeViewItem: React.FC<TreeViewItemProps> = props => {
 
     const levelPadding = level * 10;
 
+    const inputProps = {
+        defaultValue: label,
+        onSubmitInput,
+        onCancelInput,
+        submitIcon,
+        cancelIcon,
+    };
+
+    const content = mode === 'read' ? label : <TreeViewInput {...inputProps} />;
```

Comment:
I think in `write` mode we should remove the `caret` and `options` icons:

![Screenshot 2023-11-22 at 11 09 51](https://github.com/powerhouse-inc/design-system/assets/20387722/1eb10b89-7486-469a-a4c7-aa5515968ec8)

Reply:
good idea, will do